src/main/resources/static/api.yml

```diff
@@ -3310,4 +3336,37 @@ components:
           example: '#FF5733'
         withdrawn:
           type: boolean
-          example: true
\ No newline at end of file
+          example: true
+
+    CourseCreateRequest:
+      type: object
+      properties:
+        name:
+          type: string
+          example: Thinking skills programme
+        identifier:
+          type: string
+          example: 'CC'
+        description:
+          type: string
+          example: Thinking Skills Programme (TSP) description
+        alternateName:
+          type: string
+          example: 'BNM+'
+        audience:
```

Comment:
Maybe use the audience Id here instead of a string. We have the endpoint to get a list of audiences so shouldn't be a problem for the FE